packages/xchain-thorchain-query/src/thorchain-query.ts

```diff
@@ -386,6 +386,13 @@ export class ThorchainQuery {
       const destPool = await this.thorchainCache.getPoolForAsset(destAsset)
       if (!destPool.isAvailable())
         errors.push(`destinationAsset ${destAsset.ticker} does not have a valid liquidity pool`)
+      // check synth info on thornode pools
+      const pools = await this.thorchainCache.thornode.getPools()
+      const destinationAssetPool = pools.find((pool) => pool.asset === `${destAsset.chain}.${destAsset.symbol}`)
```

Comment:
getPoolForAsset() returns back type <LiquidityPool> which does not have `synth_mint_paused` since it is a MidgardPool fetch. this is why I had to get it directly from thorchain.